x-pack/plugins/cloud_security_posture/server/routes/status/status.ts

```diff
@@ -168,20 +168,32 @@ export const getCspStatus = async ({
     installedPackagePoliciesVulnMgmt,
     installedPolicyTemplates,
   ] = await Promise.all([
-    checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger),
-    checkIndexStatus(esClient, FINDINGS_INDEX_PATTERN, logger),
-    checkIndexStatus(esClient, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger),
-
-    checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger, 'cspm'),
-    checkIndexStatus(esClient, FINDINGS_INDEX_PATTERN, logger, 'cspm'),
-    checkIndexStatus(esClient, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger, 'cspm'),
-
-    checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger, 'kspm'),
-    checkIndexStatus(esClient, FINDINGS_INDEX_PATTERN, logger, 'kspm'),
-    checkIndexStatus(esClient, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger, 'kspm'),
-
-    checkIndexStatus(esClient, LATEST_VULNERABILITIES_INDEX_DEFAULT_NS, logger),
-    checkIndexStatus(esClient, VULNERABILITIES_INDEX_PATTERN, logger, VULN_MGMT_POLICY_TEMPLATE),
+    checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger, 'all', 72),
+    checkIndexStatus(esClient, FINDINGS_INDEX_PATTERN, logger, 'all', 72),
+    checkIndexStatus(esClient, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger, 'all', 72),
+
+    checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger, 'cspm', 24),
+    checkIndexStatus(esClient, FINDINGS_INDEX_PATTERN, logger, 'cspm', 24),
+    checkIndexStatus(esClient, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger, 'cspm', 24),
+
+    checkIndexStatus(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger, 'kspm', 5),
+    checkIndexStatus(esClient, FINDINGS_INDEX_PATTERN, logger, 'kspm', 5),
+    checkIndexStatus(esClient, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger, 'kspm', 5),
```

Comment:
I think kspm in practice follows the same retention as cspm (`LATEST_FINDINGS_RETENTION_POLICY`), since they share the same latest finding index and the same transform, are there other reasons to hide that data to the user if it's more than 5 hours old?

Reply:
The reason I ask this is to keep consistency between the Dashboard and the Findings page, currently, the Findings page will show cspm and kspm data for the CSPM retention period, and that might lead to inconsistencies between the Dashboard and the FIndings page.